### ISSUE TITLE
Analytic operators

### ISSUE THREAD

**Issue by NicoLaval** [InseeFr/Trevas#329]
@noahboerger you reported the following operators are not implemented:

- standard_deviation_pop
- standard_deviation_samp
- var_pop
- avg
- count
- max
- median
- min
- sum

They are with aggr shape.

The following shape is not supported: `DS_r := min ( DS_1 group by Id_1 )`

**Comment by noahboerger** [comment]
Yes that confirms to my observed behaviour. When i was transforming the test cases to use the aggr shape it was working as expected. Is it planed to also support the shape `DS_r := min ( DS_1 group by Id_1 )`?
